src/screens/runPlan/PlanItemList.tsx

```diff
@@ -37,11 +43,27 @@ export class PlanItemList extends React.PureComponent<Props, State> {
     this.setState({ planItems });
   };
 
+  handleStudentChange = (documentSnapshot: RNFirebase.firestore.DocumentSnapshot) => {
+    if(documentSnapshot.exists){
+      this.setState({ student: documentSnapshot.data() });
+    }
+  }
+
+  completedPlanItemCounter() {
+    let countCompletedPlanItems = 0;
+    for (let planItem of this.state.planItems){
+      if(planItem.completed == true)
+        countCompletedPlanItems++;
+    }
+    return countCompletedPlanItems;
+  };
+
   renderItem = ({ item, index }: { item: PlanItem; index: number }) => (
     <PlanItemListItem
       planItem={item}
       index={index}
-      textSize={this.props.student.textSize}
+      student={this.state.student}
```

Comment:
I'm wondering why should we use the student property instead of textSize? I guess textSize give us a little bit more separation between the handling settings logic and just displaying the item. What do you think about it? :)

Reply:
I agree I'll change it back to textSize and textCase